modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/routes/ContextRoutes.scala

```diff
@@ -150,27 +160,55 @@ class ContextRoutes(contextQueries: FilterQueries[Future, ContextId], base: Uri)
             val filter     = filterFrom(deprecatedOpt, None, querySettings.nexusVocBase) and publishedExpr(publishedOpt)
             implicit val _ = (id: ContextId) => contexts.fetch(id)
             (pathEndOrSingleSlash & authenticateCaller) { implicit caller =>
-              contextQueries.list(filter, pagination, None, sort).buildResponse(fields, base, prefixes, pagination)
+              sort match {
+                case SortList.Empty =>
```

Comment:
Can we sort resources based on arbitrary properties?

Reply:
I think we could, but it would require the property on which we sort to match exactly what we indexed in elastic search, e.g. `nxv:rev` and `{nexusBase}/vocabs/term/v1.0.0/rev` would not be treated as the same. We could do this like that for now and later expand the properties to full URIs when indexing.

Reply:
But for now we could do it for properties which we have in ES with full URIs, like `deprecated`, `updatedAt` etc.

Reply:
Ok, let's avoid sorting for now and handle it in a separate pull request.